components/atom/input/src/Input/Component/index.scss

```diff
@@ -65,6 +65,10 @@ $class-read-only: '#{$base-class}--readOnly';
   @each $state, $color in $states-atom-input {
     &.sui-AtomInput-input--#{$state} {
       border-color: $color;
+
+      &:focus {
+        box-shadow: 0 0 3px 0 $color;
```

Comment:
would it make sense to define this as a variable so that it can be overridden by themes?

Reply:
The part of the pixels yes, but the color don't really think so, because we want that the border-color match with the box-shadow color

Reply:
Or another approach: If a vertical needs to override this value by its team, can add this feature. It might never happen.